### PR TITLE
Implement AnnotationDisplay to the right of SnippetArea

### LINKED ISSUE #3: Display annotations to right of codemirror

### PR DESCRIPTION
**Pull request by solkaz**
Fixes #3 

This PR implements the AnnotationDisplay to the right of the SnippetArea in AppBody. It will display a simple header if no annotation is being displayed, and can be toggled on/off by clicking the displayed snippet line in the gutter

### CODE REVIEW

**Comment by dane-johnson** [comment]
Screenshots?

File: src/containers/AnnotationDisplay.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
 import MarkdownRenderer from 'react-markdown-renderer';
 import { connect } from 'react-redux';
 
+import markdownRendererOptions from '../util/markdown-renderer-options';
+
+const styles = {
+  container: {
+    flex: '1 1 auto',
```
Comment (Hopding) [inline src/containers/AnnotationDisplay.js:9]: Not a fan of CSS shorthand. I think it is only convenient when you're writing it.

**Review by Hopding — APPROVED** [review]
:shipit:

File: src/containers/AnnotationDisplay.js (hunk 1 above)
Comment (dane-johnson) [reply src/containers/AnnotationDisplay.js:9]: It's recognizable to anyone who knows web layout.

**Review by dane-johnson — APPROVED** [review]
🛥

**Comment by dane-johnson** [comment]
screen shot 2017-04-24 at 12 27 49 pm

**Pull request merged**